.github/pull_request_template.md

```diff
@@ -1,4 +1,4 @@
-## [NNN](https://dev.azure.com/VP-BD/DECD/_workitems/edit/NNN) (ADO label)
+## [AB#NNN](https://dev.azure.com/VP-BD/DECD/_workitems/edit/NNN) (ADO label)
```

Comment:
only `#ABNNN`  is required  the rest is automagically  added

`AB#NNN - ADO label  `